internal/prometheusbp/spectest/spec.go

```diff
@@ -142,22 +145,42 @@ func validateLabels(name, prefix, clientOrServer string, gotLabels map[string]st
 
 // buildLables returns a set of expected labels for the metric name provided.
 // prefix is either thrift, http, or grpc.
+func buildLables(name, prefix, clientOrServer string) map[string]struct{} {
+	var labelSuffixes = []string{}
```

Comment:
```suggestion
	var labelSuffixes []string
```

As written this conceptually (though not in practice) "allocates" a new slice, when appenad and such all work just fine on the zero value (nil) as above.